### DISCUSSION TITLE
How do I connect one link to another link?

### DISCUSSION THREAD

**Discussion by IshaanShettigar** [clientIO/joint#2397] [Q&A] [answered]
### Introduction

Currently my application connects links to elements. I need to able to connect links to other links as well. Something like this (I sketched it out in MS Paint)
![image](https://github.com/clientIO/joint/assets/77607172/b835cb78-980f-4e3c-99f6-8984f7f7090f)
![image](https://github.com/clientIO/joint/assets/77607172/f3c9c9ec-5f0a-4ec1-9324-abcc3cb647e3)

How do I do this? I have setup the links in my application to connect to ports on my elements like so. 

![image](https://github.com/clientIO/joint/assets/77607172/dd04934e-5dce-4a92-9bf4-c79e4ca04d49)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS

**Comment by kumilingus** [comment, marked as answer]
Connecting links to links is disabled by default. You can enable it by returning `true` if the connected cell is a link, from the [validateConnection](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#dia.Paper.prototype.options.validateConnection) callback.

Here are examples of [link-to-link](https://www.jointjs.com/demos?feature=Link-to-link) connection.

**Reply by IshaanShettigar** [reply]
Small follow up question. Whenever I try to connect a link to another link it automatically connects it at the center, I can move it around after that using the targetAnchorTool but I would like the default connection point to just be wherever the mouse is hovering on the link. Below is a screenshot
![image](https://github.com/clientIO/joint/assets/77607172/d05a6136-ff64-4da5-b5ab-9e890e4ce71e)
I want it to connect at the top itself but it automatically connects to the middle like so 
![image](https://github.com/clientIO/joint/assets/77607172/152de257-5cfb-4bf3-aa5e-60046a2d50f5)

**Reply by kumilingus** [reply]
See https://github.com/clientIO/joint/discussions/1986#discussioncomment-4683448.
